experimental/dds/tree2/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -990,68 +984,78 @@ describe("SharedTree", () => {
 	describe("Events", () => {
 		it("triggers events for local and subtree changes", () => {
 			const view = testTreeView();
-			const root = view.context.root.getNode(0);
+			const rootNode = view.context.root.getNode(0);
+			const root = view.root as unknown as { x: number };
 			const log: string[] = [];
-			const unsubscribe = root[on]("changing", () => log.push("change"));
-			const unsubscribeSubtree = root[on]("subtreeChanging", () => {
+			const unsubscribe = rootNode[on]("changing", () => log.push("change"));
+			const unsubscribeSubtree = rootNode[on]("subtreeChanging", () => {
 				log.push("subtree");
 			});
 			const unsubscribeAfter = view.events.on("afterBatch", () => log.push("after"));
 			log.push("editStart");
-			root[valueSymbol] = 5;
+			root.x = 5;
 			log.push("editStart");
-			root[valueSymbol] = 6;
+			root.x = 6;
 			log.push("unsubscribe");
 			unsubscribe();
 			unsubscribeSubtree();
 			unsubscribeAfter();
 			log.push("editStart");
-			root[valueSymbol] = 7;
+			root.x = 7;
 
 			assert.deepEqual(log, [
 				"editStart",
 				"subtree",
 				"change",
+				"subtree",
```

Comment:
@CraigMacomber The changes added these additional events, I think because it is now a delete and insert of the child node. Does this look right or am I doing something else wrong?

Reply:
Sounds right.